Bio/PDB/__init__.py

```diff
@@ -19,15 +19,29 @@
 - Joao Rodrigues
 - Kristian Rother
 - Eric Talevich
+- Simon Duerr
 - and many others.
 """
 
-# Get a Structure object from a PDB file
-from .PDBParser import PDBParser
+import gzip
+from pathlib import Path
+from mimetypes import guess_type
+from functools import partial
+
 
+# Get a Structure object from a PDB file using available parsers
+from .PDBParser import PDBParser
 from .MMCIFParser import MMCIFParser
 from .MMCIFParser import FastMMCIFParser
 
+try:
+    from .mmtf import MMTFParser
+except ImportError:
+    pass
+
+
+from Bio.File import as_handle
```

Comment:
Although this is not a "rule", I usually order imports as: standard lib, third-party, intra-package, intra-module. So, I'd move this up before the parsers.